.github/pr-title-checker-config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "LABEL": {
+    "name": "title needs [type]",
+    "color": "EEEEEE"
+  },
+  "CHECKS": {
+    "prefixes": ["[BugFix] ", "[BugFix]", "[Feature]", "[Refactor]", "[UT]"],
```

Comment:
There are 2 [BugFix] here, is it okay to keep only one?
I think we may also have [WIP]、[Enhancement].
and I think [Lake] should not be use for prefix it might be use for label sig/cloud

Reply:
[BugFix]/[Enhancement]/[Feature]/[Refactor]/[UT]/[Docs] must be need.. Another type like [WIP][Lake][Summer Camp] can also added by purpose.

Reply:
the following "regexp": "\\[.*\\].*", also can be used for title check, any title starts with [type] will be ok